writeengine/bulk/we_tableinfo.h

```diff
@@ -183,6 +195,7 @@ class TableInfo : public WeUIDGID
   int finishBRM();                       // Finish reporting updates for BRM
   void freeProcessingBuffers();          // Free up Processing Buffers
   bool isBufferAvailable(bool report);   // Is tbl buffer available for reading
+  int openTableFileParquet(int64_t &totalRowsParquet);
```

Comment:
Some methods in `openTableFileParquet` use totalRowsParquet as a parameter which needs `int64_t` type so I use `int64_t` here.

Reply:
See my comment in `TableInfo::readTableData()`. `int64_t` is fine, just add a comment in that function explaining why you use `int64_t` instead of `RID` for this variable.